web/virtual_lab/templates/virtual_lab/physics/circuit.html

```
@@ -0,0 +1,150 @@
+{# web/virtual_lab/templates/virtual_lab/physics/circuit.html #}
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)